Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "kurbo"
-version = "0.5.0"
+version = "0.6.0"
```

Comment:
Looks good, but I don't see why this is a semver bump...

Reply:
yea I was unsure about this, see my (revised) comment above.

Reply:
I guess this is something of a judgment call; it changes the behavior, but I'm pretty sure the contract for both Display and Debug is "human-readable" (the docs say "user-facing" which seems more ambiguous). My personal take is that code shouldn't be depending on the exact formatting, but otoh it also isn't going to hurt to be conservative here.

Reply:
I am happy here to defer to the practical reality that this will break nobody's code, and do a minor bump. This would also let us re-publish and see if the docs get fixed.